src/test/components/__snapshots__/CallNodeContextMenu.test.js.snap

```diff
@@ -12,13 +12,15 @@ exports[`calltree/CallNodeContextMenu basic rendering renders a full context men
     class="react-contextmenu-item"
     role="menuitem"
     tabindex="-1"
-    title="Merging a function removes it from the profile, and assigns its time to the function that called it. This happens anywhere the function was called in the tree."
   >
     <span
       class="react-contextmenu-icon callNodeContextMenuIconMerge"
     />
     <div
       class="react-contextmenu-item-content"
+      title="Merging a function removes it from the profile, and assigns its time to the
```

Comment:
This is one of the problems we have after moving it to the inner div. Previously we could make it one line, since localized component was in the parent. But we can't make them one line anymore here.